src/workers/newNotificationMail.ts

```diff
@@ -481,7 +481,9 @@ const notificationToTemplateData: Record<NotificationType, TemplateDataFunc> = {
       user_reputation: user.reputation,
     };
   },
-  squad_blocked: null,
+  squad_blocked: async () => {
+    return null;
+  },
```

Comment:
Just a question for my understanding, what is the difference here between async method and just value?

Reply:
It's mostly the implementation expecting a function so failed on null value itself.